tests/integration_tests/test_beam_bend_coupling.py

```diff
@@ -170,3 +173,14 @@ def setup_tacs_problems(self, comm):
             )
 
         return tacs_probs, fea_assembler
+
+    # We have to skip these tests in complex mode because the beam
+    # element uses complex step to approximate the Jacobian and this
+    # leads to issues with complex stepping the sensitivities.
+    @unittest.skipIf(TACS_IS_COMPLEX, "Skipping test_total_dv_sensitivities")
+    def test_total_dv_sensitivities(self):
+        super().test_total_dv_sensitivities()
+
+    @unittest.skipIf(TACS_IS_COMPLEX, "Skipping test_total_xpt_sensitivities")
+    def test_total_xpt_sensitivities(self):
+        super().test_total_xpt_sensitivities()
```

Comment:
How come this wasn't an issue previously?

Reply:
I'm not sure. There error I was seeing the tes was really finicky. I had a beam test that worked routinely and the modified one of the dvs to the 16th digit (ie. `t=0.01000000000000001`) and the complex sensitivity tests suddenly fail by a significant margin. To be safe I don't think we should be using cs with these elements since they are already using cs under the hood.